### ISSUE TITLE
Tripadvisor Reviews Classification using NLP

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#266]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Tripadvisor Reviews Classification using NLP</br>
:red_circle: **Aim** : Create a classification model based on NLP on the given dataset.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/arnabchaki/tripadvisor-reviews-2023</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by maithri-3** [comment]
Hi @abhisheks008 ! I would like to take up this issue, please assign it to me.

- Full name : Maithri Muralikrishna
- GitHub Profile Link : https://github.com/maithri-3
- Email ID : m.muralikrishna.cse21@itbhu.ac.in
- Approach for this Project : 
  I have recently used the LSTM technique for prediction of power usage. It was a very effective RNN technique with a good accuracy rate. 
I am keen to use it for natural language processing and detect the sentiments of the Tripadvisor reviews.
Hence, my approach is the LSTM technique using python and its libraries - numpy, pandas etc.
- What is your participant role? (Mention the Open Source program)- Social Summer of Code 2023

Thank you. Will be glad to work on this issue if assigned to me.

**Comment by abhisheks008** [comment]
I like the approach of using LSTM. Can you use 2 more deep learning methods to get a better comparison between the models, to find the best fitted one. Are you comfortable with that?

@maithri-3

**Comment by maithri-3** [comment]
@abhisheks008 yes. I have other models in my mind as well- GRU, XGBoost, Random forest etc. I will study and compare the different models.

**Comment by abhisheks008** [comment]
Cool. Issue assigned to you @maithri-3

**Comment by maithri-3** [comment]
Thanks @abhisheks008 !

**Comment by aindree-2005** [comment]
Full name : Aindree Ch
GitHub Profile Link : https://github.com/aindree-2005
Email ID : [aindree2005@gmail.com](mailto:aindree2005@gmail.com)
Approach for this Project :
Comparison of BerT vs GRU
What is your participant role? CodePeak 2023

@abhisheks008 Can I work on this issue?

**Comment by abhisheks008** [comment]
Hi @aindree-2005 it's a nice approach. I'd suggest you to try out other models for this project which will also enhance your knowledge and give you a hands-on experience on it.

Issue assigned to you.
